src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -298,6 +290,22 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
     display: none;
     visibility: hidden;
   }
+
+  .pf-m-chip-container {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: var(--pf-c-data-toolbar__expandable-content--m-chip-container--MarginTop);
```

Comment:
Sorry about this, I was reading the selector wrong before and had you update it incorrectly. I think the way you had it before was fine, though technically it should be this if we don't want to run into CSS var naming conflicts

`--pf-c-data-toolbar__expandable-content__group--m-chip-container--MarginTop`